### ISSUE TITLE
No sound coming out of BT device

### ISSUE THREAD

**Issue by ntelogo** [wastis/LinuxAddonRepo#3]
I've managed to connect two different devices via BT (JBL Charge 3, Bose Headset) but none of the available options in Audio Output gives me any sound. The devices that appear in Audio Output are all some variation of BCM2835 Headphones. 

Any clues on how to troubleshoot this?

Thnx!

**Comment by ntelogo** [comment]
Kodi v.: 19.4
Raspi OS bullseye
Linux kernel: 6.1.21
RPi 4 B

**Comment by wastis** [comment]
Sorry, I do not have much time the next 2 days, however just to point you to some directions. Have you switched Kodi to pulseaudio and installed "pulseaudio-module-bluetooth"?

Here is some example how to set-up and configure Kodi for pulseaudio: [Pi Headless Kodi](https://github.com/wastis/PulseEqualizerGui/wiki/Example-setup-on-Raspberry-Pi)

**Comment by ntelogo** [comment]
Thnx for the guidance! 
I followed your example but did not manage to make it work. Then I upgraded to Kodi 20, following [this guide](https://linuxaudiofoundation.org/install-kodi-20-1-on-debian-11-12-bullseye-bookworm/) and all problems were solved! It works for me for the time being, but I understand that this is far from an optimum solution. 

In any case.. Thnx again. I really appreciate your help!
V.
